scalafix-cli/src/main/scala/scalafix/internal/v1/Args.scala

```diff
@@ -28,7 +28,6 @@ import metaconfig.annotation._
 import metaconfig.generic.Surface
 import metaconfig.internal.ConfGet
 import metaconfig.typesafeconfig.typesafeConfigMetaconfigParser
-import pprint.TPrint
```

Comment:
`import metaconfig.{TPrint => _, _}` would have been nicer but it won't work against metaconfig 0.9.10 (last published artifact for scala 2.11)